### PR TITLE
fix101

### PR DESCRIPTION
**Pull request by vpn1997**
Fix of #101 .Now we can add labels to outputs.

### CODE REVIEW

File: api/tests.py [hunk 1]
```diff
@@ -339,18 +339,20 @@ def test_create_output_component(self):
...
-        response = self.client.put(url, json.dumps(payload),
-                                   content_type="application/json")
-        response = json.loads(response.content.decode('utf-8'))
-        response["props"] = json.loads(response["props"])
-        self.assertEqual(response["base_component_id"],
-                         payload["base_component_id"])
-        self.assertEqual(response["props"], payload["props"])
-        self.assertEqual(response["user_id"], payload["user_id"])
+        prop=[[{"id": "3", "label": ""}],[{}]]
+        for i in prop:
```
Comment (AvaisP) [inline api/tests.py]: Split into 2 test cases

**Comment by AvaisP** [comment]
@vpn1997 I have tested this locally. It works fine. However, if I try to modify the output component later, it doesn't display the stored header. Please fix that and then I will merge this.

**Comment by AvaisP** [comment]
Modifying the input component displays the label so you might find it helpful to go through the input component code

File: api/tests.py (hunk 1 above)
Comment (vpn1997) [reply api/tests.py]: I have added these changes on different PR which is on increasing coverage . :smile:

Comment (AvaisP) [reply api/tests.py]: Alright

File: webpack-stats-local.json [hunk 2]
```diff
@@ -1 +0,0 @@
-{"status":"done","chunks":{"main":[{"name":"bundle.js","publicPath":"http://localhost:3000/static/bundles/bundle.js","path":"/home/avais/Desktop/gsoc/django/post/django_server/django_server/static/bundles/local/bundle.js"}],"vendor":[{"name":"vendor.js","publicPath":"http://localhost:3000/static/bundles/vendor.js","path":"/home/avais/Desktop/gsoc/django/post/django_server/django_server/static/bundles/local/vendor.js"}]},"publicPath":"http://localhost:3000/static/bundles/"}
```
Comment (vpn1997) [inline webpack-stats-local.json]: I dont have this file on my branch .Idk why it is showing here.

File: Origami/src/components/outputcomponents/ScatterGraphOutput/ScatterGraphOutputShowcaseCard.js [hunk 3]
```diff
@@ -31,7 +31,15 @@ class ScatterGraphOutputShowcaseCard extends OutputShowcaseCard {
     } else {
       let propsToStore = [];
       this.state.headers.map(header => {
-        propsToStore.push(header);
+           if(typeof header=='object')
```
Comment (AvaisP) [inline Origami/src/components/outputcomponents/ScatterGraphOutput/ScatterGraphOutputShowcaseCard.js]: Remove extra tab

File: Origami/src/components/outputcomponents/ScatterGraphOutput/ScatterGraphOutputShowcaseCard.js [hunk 4]
```diff
@@ -31,7 +31,15 @@ class ScatterGraphOutputShowcaseCard extends OutputShowcaseCard {
...
+        {
```
Comment (AvaisP) [inline Origami/src/components/outputcomponents/ScatterGraphOutput/ScatterGraphOutputShowcaseCard.js]: Take the brace next to the condition [previous line]

**Comment by AvaisP** [comment]
Good work @vpn1997  Please fix the styling errors and then I'll merge this

**Comment by AvaisP** [comment]
@vpn1997 Please pass all the modified files through prettier

**Comment by vpn1997** [comment]
@AvaisP done.

**Comment by vpn1997** [comment]
@AvaisP hey i have modified files through prettier.

**Comment by AvaisP** [comment]
Great job! @vpn1997

**Pull request merged**